README.md

```diff
@@ -159,38 +172,227 @@ function App() {
 export default App;
 ```
 
-You can pass Bridge options as a third parameter as well:
+`StoryblokComponent` renders the route components dynamically, using the list of components loaded during the initialization inside the `storyblokInit` function. 
+
+This is how you can pass the Bridge options as a third parameter to `useStoryblok`:
 
 ```js
-useStoryblok(story.id, (story) => (state.story = story), {
+useStoryblok(story.id, {version: 'draft'}, {
```

Comment:
I think the example is a bit misleading, as currently you need to pass `resolveRelations` param twice (in 2nd and 3rd parameter)

Reply:
@alexjoverm good point.
@arorachakit you think we should add a clarification about that? Or add the params twice?

Reply:
I think we can add the params twice. Or we can even use other bridge params, maybe `preventClicks`?

Reply:
Totally agree @arorachakit